### PR TITLE
this should fix #4

### LINKED ISSUE #4: Mac OS can't find suitable image
``` Julia
julia> using FreeType

julia> library = Array(FT_Library, 1)
1-element Array{Ptr{FreeType.FT_LibraryRec_},1}:
 Ptr{FreeType.FT_LibraryRec_} @0x0000000103b3fdf0

julia> error = FT_Init_FreeType(library)
dlerror loading '': dlopen(/Users/epn/.julia/v0.4/Homebrew/deps/usr/lib/, 5): no suitable image found.  Did find:
    /Users/epn/.julia/v0.4/Homebrew/deps/usr/lib/: not a file
ERROR: error compiling FT_Init_FreeType: could not load library
```

### PR DESCRIPTION
**Pull request by SimonDanisch**

### CODE REVIEW

**Comment by SimonDanisch** [comment]
Sorry, I mean #4 ...

File: src/FreeType.jl [hunk 1]
```diff
@@ -3,13 +3,18 @@ module FreeType
...
     [
         "/usr/lib/i386-linux-gnu", "/usr/lib/x86_64-linux-gnu",
-        Pkg.dir("WinRPM", "deps", "usr", "$(Sys.ARCH)-w64-mingw32", "sys-root", "mingw", "bin")
+        Pkg.dir("WinRPM", "deps", "usr", "$(Sys.ARCH)-w64-mingw32", "sys-root", "mingw", "bin"),
+        "/Developer/SDKs/MacOSX10.7.sdk/usr/X11/include/freetype2", "/usr/X11/lib/"
     ]
 )
 
+if isempty(freetype)
+    error("FreeType library could not be found. Please file a Github issue with details about your operating system")
```
Comment (jhasse) [inline src/FreeType.jl]: Good idea! :) I would change it too

``` julia
error("FreeType library could not be found. Please file an issue here: https://github.com/jhasse/FreeType.jl/issues/new")
```

though (So people would know _where_ to file the issue).

**Comment by jhasse** [comment]
Can you change the error message and fix up the commit message? Please also add what this does to the commit message, e. g. `Find FreeType on Mac OS X. Fixes #4`

**Comment by jhasse** [comment]
Ah you are faster than me :)
Would be great if that wrong commit message goes away though. Use

```
git reset --soft HEAD^
git commit --amend -m 'Find FreeType on Mac OS X. Fixes #4'
git push --force
```

for that (not tested).

**Comment by SimonDanisch** [comment]
hope this did it...

**Comment by jhasse** [comment]
Somehow 42487f2fcfb99f0833a0bbb603b5b3ba1479469c got mixed up in your commit. But that shouldn't be a problem.

Thanks :)

**Pull request merged**